scala/tempo/src/main/scala/com/databrickslabs/tempo/TSDF.scala

```diff
@@ -165,44 +138,54 @@ sealed trait TSDF
  * @param tsColumn timeseries column
  * @param partitionCols partitioning columns
  */
-private[tempo] sealed class BaseTSDF(val df: DataFrame,
-                                     val tsColumn: StructField,
-                                     val partitionCols: Seq[StructField] )
+private[tempo] sealed class BaseTSDF(val df: DataFrame, val schema: TsStructType )
 	extends TSDF
 {
```

Comment:
Please follow Databricks Scala Styleguide coding and formatting practices - https://github.com/databricks/scala-style-guide